db/migrate/20190617191109_move_awx_credentials_to_authentications.rb

```diff
@@ -0,0 +1,142 @@
+class MoveAwxCredentialsToAuthentications < ActiveRecord::Migration[5.0]
+  class Authentication < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options
+  end
+
+  class MiqDatabase < ActiveRecord::Base; end
+
+  ENCRYPTED_ATTRIBUTES = %w[
+    auth_key
+    auth_key_password
+    become_password
+    password
+  ].freeze
+
+  FIELD_MAP = {
+    'authorize_password' => 'become_password',
+    'become_username'    => 'become_username',
+    'become_password'    => 'become_password',
+    'password'           => 'password',
+    'secret'             => 'auth_key',
+    'security_token'     => 'auth_key',
+    'ssh_key_data'       => 'auth_key',
+    'ssh_key_unlock'     => 'auth_key_password',
+    'username'           => 'userid',
+    'vault_password'     => 'password'
+  }.freeze
+
+  OPTIONS_FIELDS = %w[
+    authorize
+    become_method
+    client
+    domain
+    host
+    project
+    subscription
+    tenant
+  ].freeze
+
+  def up
+    embedded_ansible_authentications.each do |auth|
```

Comment:
Should this be using `find_each` to work in batches?

Reply:
I'm expecting a maximum of something like 10 or 20 of these, but sure.

Reply:
Looks like the default batch size is 1000, for `find_each` so I really don't think it's necessary.